dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -216,4 +223,13 @@ http.compression.types=List of content types for response compression
 http.compression.types.desc=To include a content type in addition to the default values, affix the add (+) character as a prefix to that content type. To exclude a content type from compression, affix the remove (-) character as a prefix to that content type. Note: The wildcard (*) character is supported only as a content subtype, such as text/*.
 
 http.compression.serverPreferredAlgorithm=Server-preferred algorithm for compression
-http.compression.serverPreferredAlgorithm.desc=The configured compression algorithm is used to compress the body of responses when it is specified with a non-zero quality value in the request's Accept-Encoding header. The valid compression algorithms include: deflate, gzip, x-gzip, zlib, and identity.
\ No newline at end of file
+http.compression.serverPreferredAlgorithm.desc=The configured compression algorithm is used to compress the body of responses when it is specified with a non-zero quality value in the request's Accept-Encoding header. The valid compression algorithms include: deflate, gzip, x-gzip, zlib, and identity.
+
+http.cookies.samesite.lax=List of samesite=lax cookies
+http.cookies.samesite.lax.desc=List of cookie names or patterns that should have the SameSite attribute set to a value of "Lax", if not already defined. A single wildcard (*) character is supported as a standalone value, or following a cookie name prefix. Any cookie name or pattern defined by this list must be unique and not present in the 'sameSiteNone' nor 'sameSiteStrict' configurations.
```

Comment:
The use of "single" in the second sentence makes me think people might think they only get one wildcard used with a prefix.  What about this (note that I have left the asterisks out only because this editor thinks I mean that the text between them is supposed to be italicized):

A single wildcard ( ) character is supported as a standalone value, or following cookie name prefixes.
or
A single wildcard ( ) character is supported as a standalone value. Any number of cookie name prefixes can have a wildcard as a final character.

(If you think is a good change, make the same edit for the other two .desc's also)